### ISSUE TITLE
Styling and extending the search bar

### ISSUE THREAD

**Issue by fnielsen** [fnielsen/scholia#1460]
Styling and extending the search bar in the menu and on the front page. Make it more similiar the search bar on Wikidata. 
* Add borders to the items.
* Add "more" option, so items beyond 10 can be selected.

**Comment by carlinmack** [comment]
Can the new search be added to this page https://scholia.toolforge.org/search? @DanielFLopez

**Comment by DanielFLopez** [comment]
> Can the new search be added to this page https://scholia.toolforge.org/search? @DanielFLopez

Do you mean the select2 plugin? We think that the advanced search has some special characteristics, such as the identification of Doi and arXiv, this requires certain validations and handling of special cases, so it should be kept on the backend side, with the plugin all these cases should be moved to the font. @carlinmack

**Comment by carlinmack** [comment]
I think you're misunderstanding me. The search with autocomplete is already available on the home page and every other page. This search box https://scholia.toolforge.org/search does not have the autocomplete.

**Comment by DanielFLopez** [comment]
Yes, that's because Select2 can also retrieve results from other data sources, like a remote JSON API, the search from the main page works that way, but the advanced search processes the text in the backend to detect the Doi, arXiv codes and it redirects them to their pages, why then what would be the difference between the home search and advance search?

**Comment by carlinmack** [comment]
I guess I kind of understand, however I still think that all search boxes should be as powerful as they can be. As in, I don't see why the search results page search box doesn't have the autocomplete. I think at this point it would be easier for me to make the PR myself?

**Comment by carlinmack** [comment]
@fnielsen can this be closed? or anything else to be done

**Comment by egonw** [comment]
@carlinmack, one thing I don't like of the new search functionality is that I have to click twice before I can start typing. First click is to get the new UI, resulting in (without cursor focus):

![image](https://user-images.githubusercontent.com/26721/131218486-b183c249-71e4-4ae1-ad21-ec85fb42d0f9.png)

and then a second click is needed to give focus to the second search so that I can type things:

![image](https://user-images.githubusercontent.com/26721/131218507-afd60054-9dc6-4afb-83b1-b4395ebd590e.png)

**Comment by carlinmack** [comment]
Yeah I think it's due to the plugin being used (select2) is meant to be used for dropdowns rather than search input. I would suggest instead using the HTML <datalist> element to provide the autocomplete
